Manifests/ManagedPreferencesApplications/menu.nomad.login.ad.plist

```diff
@@ -1288,6 +1405,18 @@
 			<key>pfm_value_placeholder</key>
 			<string>_nomadlogin</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>1.4</string>
+			<key>pfm_description</key>
+			<string>Message to show when the password needs to sync</string>
+			<key>pfm_name</key>
+			<string>MessagePasswordSync</string>
+			<key>pfm_title</key>
+			<string>Message Password Sync</string>
```

Comment:
Very nit-picky, but could this be `Password Sync Message` instead?

Reply:
Sure. Changed